PayPal/src/main/java/com/braintreepayments/api/PayPalVaultRequest.java

```diff
@@ -0,0 +1,82 @@
+package com.braintreepayments.api;
+
+import android.text.TextUtils;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class PayPalVaultRequest extends PayPalRequest {
```

Comment:
A doc string would be nice on this class too.

Reply:
Added 👍